src/__tests__/functions.test.ts

```diff
@@ -42,6 +42,38 @@ const mockLocations = [
   },
 ];
 
+const mockCollectInputsReturns = [
+  {
+    skipped: false,
+    text: '_text',
+    toggles:[],
```

Comment:
Can we include tests where toggles are used as well so we can test the toggle mapping code